magento/magento2
# Fix widget recent view product not fit in mobile view

### Description (*)

Fix for widget recent view product fit again in mobile view

### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#27058: Recently viewed products in Luma theme does not fit on screen

### Manual testing scenarios (*)

1. Magento 2.3.4+ with sampledata 
2. Theme blank or luma
3. Add a new widget for "Recently viewed products"
4. Select Layout update "All pages" and container "Sidebar Main"
5. View some product in frontend
6. Visit some products in mobile view until the widget appears in mobile view

### Questions or comments


### Contribution checklist (*)
